### PR TITLE
improve-styling-of-code-snippets

### LINKED ISSUE #118: [UDPATE] Improve Styling of Code Snippets
### Problem
The Code Snippets we provide in the blogs are dull and the colors are not very readable. 
Screenshot 2024-10-16 at 4 22 35 PM

we can change it to some good themes from prism.
reference - [Prism](https://prismjs.com/)

### PR DESCRIPTION
**Pull request by iamsmruti**
### Describe the Problem
The code snippets in the blogs are not up to the mark in terms of styling. 

### Solution
Used Prism to add some styling to the code snippets. 

Screenshot 2024-10-17 at 3 52 45 PM

Resolves #118

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -52,7 +52,8 @@ links:
 # http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 timezone:    Asia/Kolkata
 future:      true
-highlighter: pygments
+
```
Comment (manishElitmus) [inline _config.yml:55]: Remove space

File: _devconfig.yml [hunk 2]
```diff
@@ -52,7 +52,8 @@ links:
 # http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 timezone:    Asia/Kolkata
 future:      true
-highlighter: pygments
+
```
Comment (manishElitmus) [inline _devconfig.yml:55]: Remove extra line

File: _includes/head.html [hunk 3]
```diff
@@ -83,4 +83,7 @@
 <link rel="stylesheet" type="text/css" href="{{ site.url }}{{ site.baseurl }}/assets/css/font-elitmuslogo.css" />
 
 <link rel="stylesheet" href="{{ site.url }}{{ site.baseurl }}/assets/css/customize.css">
-<link rel="stylesheet" href="{{ site.url }}{{ site.baseurl }}/assets/css/forkit.css">
\ No newline at end of file
+<link rel="stylesheet" href="{{ site.url }}{{ site.baseurl }}/assets/css/forkit.css">
+
+<!-- Prism -->
+<link rel="stylesheet" href="{{ site.url }}{{ site.baseurl }}/assets/css/prism.css">
```
Comment (manishElitmus) [inline _includes/head.html:89]: Add new line

File: _config.yml (hunk 1 above)
Comment (iamsmruti) [reply _config.yml:55]: I added a line to separate out the code snippet features that is markdown and highlighter.

**Review by manishElitmus — APPROVED** [review]

**Pull request merged**